src/modules/tests/test-data/test-data.selector.ts

```diff
@@ -72,6 +72,9 @@ export const getDangerousFaultSummaryCount = (data: TestData): number => {
 
 export const sumVehicleCheckFaults = (vehicleChecks: VehicleChecks): number => {
 
+  if (!vehicleChecks || !vehicleChecks.showMeQuestion || !vehicleChecks.tellMeQuestion) {
```

Comment:
Is this guarding against a newly introduced NPE, or was it a speculative fix? Do we still need it?

Reply:
it was created to guard against an issue during dev - however upon dev completion I don't think the situation is happening so can remove this guard clause.